templates/birds/utility.macros.j2

```diff
@@ -5,12 +5,14 @@
     <div class="h-card">
       {{ utility.icon(page.config.bird, size='face') }}
 
-      {{ page.render_summary() }}
+      <div class="bio-short">
```

Comment:
eh, IDK. I feel like maybe we should have an explicit link to read full bio to give them reason to go to birds page if they really want to learn more. Otherwise we can surface like-content faster which is more likely what they will be interested in IMO.